packages/workers/Dockerfile

```diff
@@ -38,13 +38,14 @@ COPY $SRC_CORE_PATH/package.json $DEST_CORE_PATH/
 COPY $SRC_WORKERS_PATH/package.json $DEST_WORKERS_PATH/
 COPY tsconfig* $APP_PATH/
 COPY $SRC_CLI_PATH $DEST_CLI_PATH/
-RUN pnpm install --include-workspace-root --frozen-lockfile --filter=workers... --filter=cli...
 
 COPY nx.json tsconfig* jest* babel* .eslintrc* .prettier* .eslintrc* $APP_PATH/
 COPY $SRC_CORE_PATH $DEST_CORE_PATH/
 COPY $SRC_WORKERS_PATH $DEST_WORKERS_PATH/
 RUN chmod +x $DEST_WORKERS_PATH/scripts/*.sh
 
+RUN pnpm install --include-workspace-root --frozen-lockfile --filter=workers... --filter=cli...
```

Comment:
So in my experience, if I don't move it, it builds successfully but it has the wrong modules and can not find the module serverless. I thought it was because `COPY $SRC_WORKERS_PATH $DEST_WORKERS_PATH/` overwrites the node_modules, but that's false because it's in .dockerignore. Regarding the cache, yeah it's bad from a docker pov but I belive it gets cached by `nx` instead and the docker build command doesn't get triggered. Which also means you also need to `pnpm nx clear-cache` in between tests.

Reply:
I was able to reproduce right now. 

1. Revert position
2. pnpm nx clear-cache
3. pnpm nx run workers:compose-build-image
4. pnpm saas up
5. docker logs saas_{PROJECT_NAME}-workers-1

```
S C:\Users\me\saas_remote> docker logs saas_remote-workers-1

> nx run workers:serve

Local trigger server listening at http://0.0.0.0:3005
stdout:
> workers@2.3.0 sls /app/packages/workers
> sls "invoke" "local" "-f" "SynchronizeContentfulContent" "-d" "{\"source\":\"backend.contentfulSync\",\"detail-type\":\"complete\",\"detail\":{\"id\":\"d76db14f253e4586a016f878cb75a396\",\"type\":\"complete\"}}"
stderr: node:internal/modules/cjs/loader:1080
  throw err;
  ^
Error: Cannot find module '/app/packages/workers/node_modules/serverless/bin/serverless.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
Node.js v18.17.1
stdout:  ELIFECYCLE  Command failed with exit code 1.
child process exited with code 1
```